_troubleshoot/security-admin.md

```diff
@@ -6,7 +6,7 @@ nav_order: 10
 
 # securityadmin.sh Troubleshooting
 
-This page includes troubleshooting steps for `securityadmin.sh`.
+This page includes troubleshooting steps for `securityadmin.sh`. The script can be found at `plugins/opensearch-security/tools/securityadmin.sh`. For more information about using this tool, see [Applying changes to configuration files]({{site.url}}{{site.baseurl}}/security/configuration/security-admin/).
```

Comment:
I can't find the file. Can we give a little more context?

Reply:
Never mind if it is part of the distribution and users know where to find it. :)

Reply:
This is the last part of the path for the script/tool in any of the OpenSearch installations (not the path in the security repository). the full path should vary from installation to installation. I've asked the security team for the full path for a Docker installation so we can at least provide an example. When I get that, I'll revise the doc update.